scripts/sct_straighten_spinalcord.py

```diff
@@ -952,7 +952,7 @@ def get_parser():
                                   'To keep native resolution, set this option to 0.\n',
                       mandatory=False,
                       default_value=0,
-                      deprecated=True)
+                      deprecated_by='-speed_factor')
```

Comment:
You cannot use `deprecated_by` argument here as `-speed_factor` does not have the same behavior as `-resample`.